src/lib/storages/filters.ts

```diff
@@ -71,6 +77,8 @@ export default class FiltersStorage {
       this.rootScope.broadcast('filter_delete', this.filters[update.id]);
       delete this.filters[update.id];
     }
+
+    this.appStateManager.pushToState('filters', this.filters);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/MikeOwino/tweb/pullRequest?prid=7329411)
```suggestion
    this.appStateManager.pushToState("filters", this.filters);
```